components/search/src/types.rs

```diff
@@ -7,13 +7,14 @@
 use serde::Deserialize;
 
 /// The list of possible application names that are currently supported.
-#[derive(Debug, uniffi::Enum)]
+#[derive(Clone, Debug, Deserialize, PartialEq, uniffi::Enum)]
+#[serde(rename_all = "kebab-case")]
 pub enum SearchApplicationName {
     Firefox = 1,
     FirefoxAndroid = 2,
-    FirefoxIOS = 3,
+    FirefoxIos = 3,
```

Comment:
Yeah, [Rust's convention for acronyms](https://rust-lang.github.io/api-guidelines/naming.html) leads to them looking a little funky...